modules/cluster-logging-rn-5.2.12.adoc

```diff
@@ -0,0 +1,30 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-2-12"]
+= Logging 5.2.12
```

Comment:
It looks like we changed the product name to "Logging." I think the team wanted to differentiate cluster logging from other types of logging in kubernetes. Was this name change reviewed by the appropriate stakeholders?

Reply:
Yes, Logging was approved as an acceptable short name.